### PR TITLE
List branch

### PR DESCRIPTION
**Pull request by ruchiragoswami**
"List" component is here. Please review.

### CODE REVIEW

File: lists/lists.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <h1> Lists </h1>
+
+    <h2> Simple List </h2>
+    <section class="container-for-lists">
+        <ul class="list-general list-simple ">
+            <li> India </li>
+            <li> Canada </li>
+            <li> Nepal</li>
+            <li> Sri Lanka </li>
+            <li> Maldieves </li>
```
Comment (gadgetfather) [inline lists/lists.html:22]: Hey! for these list items, you can add a slight border like this.  The rest of the code is good 👍 
image

**Review by gadgetfather — APPROVED** [review]
Good to merge!

**Review by Husain01 — APPROVED** [review]
Everything looks good. Nice work

File: lists/lists.html (hunk 1 above)
Comment (ruchiragoswami) [reply lists/lists.html:22]: I did it and it looks great. Thank you for the suggestion :) 💯

**Pull request merged**